code/modules/hydroponics/seed_datums.dm

```diff
@@ -212,7 +212,6 @@
 	name = "bluetomato"
 	seed_name = "blue tomato"
 	display_name = "blue tomato plant"
-	mutants = list("bluespacetomato")
 	chems = list(/datum/reagent/nutriment = list(1,20), /datum/reagent/lube = list(1,5))
 
 /datum/seed/tomato/blue/New()
```

Comment:
The trait can still be randomly mutated, and bluespace tomatoes themselves are un-alluring and rarely seen anyway.

Reply:
It doesn't.